projects/aws/eks-a-snow-admin-image/README.md

```diff
@@ -0,0 +1,2 @@
+# Snow EKS-A Admin image
```

Comment:
Sooo I'm literally working on the exact same task to create an EKS-A Admin machine OVA for our lab. Rather than duplicating or making a lab different eks-a admin image, what do you think about just having a generic admin image we all share and create for each cloud similar to image builder. I'm looking at all prereqs you have baked in and they are identical to mine. Only difference in what I have started is I include kind, yq, jq, golang, python, and aws ssm agent.

Reply:
I'm ok sharing the project and reusing as much code as possible

However, I would like to avoid adding more stuff to the snow image, it's already too big. I would vote to make the provisioners for the installation of those other components you have conditional to the ova `source`. Since they are just references to bash scripts, we can move them to the shared provisioners later on if we see the need.

Reply:
Agreed!